sourced/ml/transformers/repo2weighted.py

```diff
@@ -1,16 +1,28 @@
+import operator
+
 from sourced.ml.transformers import Transformer
 
 
-class Repo2WeightedSet(Transformer):
-    def __init__(self, extractors, **kwargs):
+class Documents2BOW(Transformer):
+    def __init__(self, extractors, document_column="blob_id", **kwargs):
         super().__init__(**kwargs)
         self.extractors = extractors
+        self.document_column = document_column
 
     def __call__(self, rows):
-        return rows.map(self.process_row)
```

Comment:
The code was so simple, so innocent. Why did you touch it at all?

Reply:
because I need to merge all document parts from different rows together.